### ISSUE TITLE
pip wheel lacks some LCM message inter-operability

### ISSUE THREAD

**Issue by jwnimmer-tri** [RobotLocomotion/drake#15957]
As see in the [pip-drake.patch](https://github.com/RobotLocomotion/pip-drake/blob/master/image/pip-drake.patch), we currently disable `BindCppSerializers` during wheel building.

That means that LCM-message-valued System ports lack certain features.

We should work to restore that support.

**Comment by jwnimmer-tri** [comment]
The first part of the solution here is that where we currently copy only the `/opt/drake/lib/python*/site-packages/pydrake/*` python files into the wheel, we should also copy the `site-packages/drake/*` code-generated files, which will be like so:

```
drake/__init__.py
drake/experimental_lcmt_deformable_tri_meshes_init.py
drake/experimental_lcmt_deformable_tri_meshes_update.py
drake/experimental_lcmt_deformable_tri_mesh_init.py
drake/experimental_lcmt_deformable_tri_mesh_update.py
drake/experimental_lcmt_deformable_tri.py
drake/lcmt_acrobot_u.py
drake/lcmt_acrobot_x.py
drake/lcmt_acrobot_y.py
drake/lcmt_allegro_command.py
drake/lcmt_allegro_status.py
drake/lcmt_call_python_data.py
drake/lcmt_call_python.py
... etc ...
drake/lcmt_viewer_draw.py
drake/lcmt_viewer_geometry_data.py
drake/lcmt_viewer_link_data.py
drake/lcmt_viewer_load_robot.py
```

The second part is to remove the `BindCppSerializers` nerfing from `pip-drake.patch`.

After that -- maybe everything just works?  Probably the `import pydrake.all` test is sufficient on its own, but we could also test for `from drake import lcmt_viewer_draw` to be sure.

**Comment by jwnimmer-tri** [comment]
This fix was released in `drake==0.36.0`.
